src/sizeMapping.js

```diff
@@ -125,12 +125,16 @@ function evaluateSizeConfig(configs) {
     ) {
       let ruleMatch = false;
 
-      try {
-        ruleMatch = getWindowTop().matchMedia(config.mediaQuery).matches;
-      } catch (e) {
-        logWarn('Unfriendly iFrame blocks sizeConfig from being correctly evaluated');
-
-        ruleMatch = matchMedia(config.mediaQuery).matches;
+      if (config.mediaQuery === '') {
+        ruleMatch = true;
```

Comment:
can you add a TODO here to remove this eventually?

Reply:
I can, what would you like it to say? So you can know it's purpose

Reply:
I'll add the comment in a separate PR. Accepting this change to master for now! @benjaminclot Can you create a similar PR for **prebid-2.0** as well?

Reply:
Here you go @Fawke: #4760